File: src/main/java/com/gmail/borlandlp/minigamesdtools/util/Glowing.java
package com.gmail.borlandlp.minigamesdtools.util;

import org.bukkit.Bukkit;
import org.bukkit.entity.Entity;
import org.bukkit.entity.Player;
import org.inventivetalent.glow.GlowAPI;

import java.util.ArrayList;
import java.util.Collection;
import java.util.List;

public class Glowing {
    public static void addGlowEffect(Entity entity, String colorName) {
        if (entity != null) {
            GlowAPI.Color glowColor = GlowAPI.Color.valueOf(colorName);
            Collection<Player> players = new ArrayList<>(Bukkit.getServer().getOnlinePlayers());
            GlowAPI.setGlowing(entity, glowColor, players);
        }
    }

    public static void removeGlowEffect(Entity entity) {
        if (entity != null) {
            Collection<Player> players = new ArrayList<>(Bukkit.getServer().getOnlinePlayers());
            GlowAPI.setGlowing(entity, null, players);
        }
    }

    public static List<String> getAvailableColors() {
        List<String> colors = new ArrayList<>();
        for(GlowAPI.Color color : GlowAPI.Color.values()) {
            colors.add(color.name());
        }

        return colors;
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/util/Glowing.kt
package com.gmail.borlandlp.minigamesdtools.util

import org.bukkit.Bukkit
import org.bukkit.entity.Entity
import org.bukkit.entity.Player
import org.inventivetalent.glow.GlowAPI
import java.util.*

object Glowing {
    fun addGlowEffect(entity: Entity, colorName: String = "") {
        val glowColor = GlowAPI.Color.valueOf(colorName)
        val players: Collection<Player> = ArrayList(Bukkit.getServer().onlinePlayers)
        GlowAPI.setGlowing(entity, glowColor, players)
    }

    fun removeGlowEffect(entity: Entity) {
        val players: Collection<Player> = listOf(Bukkit.getServer().onlinePlayers) as Collection<Player>
        GlowAPI.setGlowing(entity, null, players)
    }

    @JvmStatic
    val availableColors: List<String>
        get() {
            return mutableListOf<String>().apply {
                for (color in GlowAPI.Color.values()) {
                    this.add(color.name)
                }
            }
        }
}